util/file.py

```diff
@@ -473,3 +494,33 @@ def line_count(infname):
 def touch(fname, times=None):
     with open(fname, 'a'):
         os.utime(fname, times)
+
+def touch_empty(fname):
+    '''Make each file in fnames a zero-length file with the current timestamp.'''
+    with open(fname, 'w'):
```

Comment:
You're right about the name.  Reason I wanted a make-file-empty function, rather than just touch, is to ensure an empty output file even if the previous operation failed midway and left a partially-written file, or there was already a file with that name there.